samples/Uno.Toolkit.Samples/Uno.Toolkit.Samples.Shared/App.xaml

```diff
@@ -42,7 +43,29 @@
 			</ResourceDictionary.MergedDictionaries>
 
 			<x:Double x:Key="DesktopAdaptiveThresholdWidth">720</x:Double>
-			
-		</ResourceDictionary>
+
+			<!-- Note: This style is defined here, rather than in a separate file (eg Styles/Application/Splashscreen.xaml)
+			because it depends on DefaultExtendedSplashScreen which is in ToolkitResources and resources need to be
+			merged so that it can be referenced - this seems to be an issue on some platforms -->
```

Comment:
whaaat, weird. This means we can't reference ResourceDictionaries that have references to any Material/Toolkit resources in subsequent files within the MergedDictionaries for App.xaml?

Like Styles/Button.xaml can't contain a style that is based on MaterialTextButtonStyle, say?

Reply:
That does seem to be an issue - let's verify